desktop/sources/scripts/client.js

```diff
@@ -160,6 +186,21 @@ function Client () {
     this.update()
   }
 
+  this.toggleLink = () => {
+    if (this.clock.isLinkEnabled) {
+      this.link.disable()
+      this.link.disableStartStopSync()
+    } else {
+      this.link.enable()
+      this.link.enableStartStopSync()
+      this.clock.setSpeed(this.link.getTempo(true), this.link.getTempo(true), true)
+      if (!this.link.isPlaying()) {
+        this.clock.stop(false, true)
+      }
+    }
+    this.clock.isLinkEnabled = !this.clock.isLinkEnabled
```

Comment:
This is a potential fun bug if it ever gets flipped, better to set it explicitly in the if/else.